nleis/nleis.py

```diff
@@ -747,105 +746,6 @@ def __init__(self, circuit='', **kwargs):
                              ' must be equal to ' +
                              f'the circuit length ({circuit_len})')
 
-    def fit(self, frequencies, impedance, bounds=None,
```

Comment:
These all depend on _circuit_fit_ and _build_circuit_, which should be restored. We can not inherit the method directly for fitting and predict. Instead, We will need to overwrite it